### DISCUSSION TITLE
Please share condolence articles on Bram and his work

### DISCUSSION THREAD

**Discussion by chrisbra** [vim/vim#12742] [General]
Let me start, I'd like to collect some articles about Bram:
- https://www.heise.de/news/Zum-Tode-Bram-Moolenaars-Vim-Koryphaee-Weltenbummler-und-Spendensammler-9235897.html
- https://www.golem.de/news/vim-bram-moolenaar-ist-gestorben-2308-176512.html
- Hackernews Comments: https://news.ycombinator.com/item?id=37011324

Please share other articles

**Comment by kuator** [comment]
One from the japanese vim community: https://vim-jp.org/blog/2023/08/05/Vim-boss-passed-away.html

**Reply by k-takata** [reply]
Some Japanese news:
https://forest.watch.impress.co.jp/docs/serial/yajiuma/1522338.html
https://news.mynavi.jp/techplus/article/20230807-2744084/
(The second one has a mistake, though.)

**Reply by k-takata** [reply]
Another one:
https://gigazine.net/news/20230809-vim-creator-bram-moolenaar-passed-away/

**Comment by mrdubya** [comment]
From Stackdiary: https://stackdiary.com/a-tribute-to-bram-moolenaar/

**Comment by gu-fan** [comment]
A video for bram at bilibili (CHINESE) https://www.bilibili.com/video/BV1Xh4y1F7nD/?vd_source=b9d02f29e78f12e44d2aed4c83c35e0d

**Comment by jgb** [comment]
From a large Dutch IT related website:

https://tweakers.net/nieuws/212392/bram-moolenaar-ontwikkelaar-van-teksteditor-vim-is-overleden.html

**Comment by RestorerZ** [comment]
This is not an article, but a news article on a Russian IT site:
[Скончался Брам Моленаар — Отец VIM](https://habr.com/en/news/752706/)  

[And this is a section of the same site entirely dedicated to Vim](https://habr.com/en/hub/vim/)

**Reply by tsunamaru** [reply]
+ OpenNET: [Умер автор и ключевой разработчик Vim](https://opennet.ru/59557-vim)
+ LINUX.ORG.RU: [Умер Брам Моленар — создатель редактора vim](https://www.linux.org.ru/news/opensource/17315462)

**Comment by savchenko** [comment]
Not really an article, but an extensive threat at [/r/vim](https://old.reddit.com/r/vim/): https://old.reddit.com/r/vim/comments/15iunt4/bram_moolenaar_creator_of_vim_has_died/

**Reply by arbitrary-dev** [reply]
*thread

**Comment by mrdubya** [comment]
One from The Register: https://www.theregister.com/2023/08/07/bram_moolenaar_obituary/

**Reply by lproven** [reply]
Thanks for the link -- that is my article. Although I am not a Vim user myself, I tried to convey the affection and respect that the tool and its author inspired.

**Comment by yegappan** [comment]
Some more articles:

https://thenewstack.io/bram-moolenaar-author-of-the-open-source-vim-code-editor-has-died/
https://fagenwasanni.com/ai/bram-moolenaar-creator-of-the-open-source-vim-code-editor-passes-away/160833/
https://thenextweb.com/news/dev-world-mourns-loss-of-vim-creator-bram-moolenaar
https://news.itsfoss.com/vim-creator-passed-away/
https://www.hackread.com/vim-text-editor-linux-bram-moolenaar-passes-away/
https://news.slashdot.org/story/23/08/05/1632219/vims-creator-bram-moolenaar-dies-at-age-62
https://arstechnica.com/gadgets/2023/08/bram-moolenaar-creator-of-the-beloved-vim-text-editor-has-passed-away/
https://citylife.capetown/uncategorized/dutch-software-developer-bram-moolenaar-passes-away-at-62/332345/
https://lwn.net/Articles/940551/
https://www.georgevreilly.com/blog/2023/08/07/BramMoolenaarRIP.html
https://j11g.com/2023/08/07/the-legacy-of-bram-moolenaar/
https://tomaszs2.medium.com/bram-moolenaar-the-creator-of-vim-died-6451cf2f3f0a
https://lobste.rs/s/yrlb5m/message_from_family_bram_moolenaar_he
https://isp.page/news/vim-creator-bram-moolenaar-dies-aged-62/
https://debugpointnews.com/vim-bram-moolenaar/
https://www.linuxadictos.com/en/murio-bram-moolenaar-creador-de-vim.html
https://news.snbc13.com/bram-moolenaar-vim-text-editor-founder-has-died/
https://ubunlog.com/en/vim-bram-moolenaar/amp/
https://www.fskhub.com/news-cause-of-death/bram-moolenaar-death-the-creator-of-vim
https://www.reddit.com/r/linux/comments/15iv93s/bram_moolenaar_creator_of_vim_has_died/
https://www.reddit.com/r/programming/comments/15ivsje/bram_moolenaar_creator_of_vim_has_died/
https://www.reddit.com/r/opensource/comments/15j1tf2/the_creator_of_vim_bram_moolenaar_has_passed_away/
https://www.reddit.com/r/neovim/comments/15iuujn/bram_moolenaar_creator_of_vim_has_died/
https://www.reddit.com/r/archlinux/comments/15jprul/bram_moolenaar_the_creator_of_vim_has_died_rip/
https://lemmy.world/post/2712042

**Comment by chrisbra** [comment]
Some more from [George Reilly](https://groups.google.com/g/vim_dev/c/RPc6tIFI9iE/m/u3EhkdNmBQAJ):

I wrote a post about Bram at my blog: https://www.georgevreilly.com/blog/2023/08/07/BramMoolenaarRIP.html

Some more articles:
https://thenewstack.io/bram-moolenaar-author-of-the-open-source-vim-code-editor-has-died/
https://fagenwasanni.com/ai/bram-moolenaar-creator-of-the-open-source-vim-code-editor-passes-away/160833/
https://thenextweb.com/news/dev-world-mourns-loss-of-vim-creator-bram-moolenaar
https://news.itsfoss.com/vim-creator-passed-away/
https://www.hackread.com/vim-text-editor-linux-bram-moolenaar-passes-away/

**Comment by dpelle** [comment]
A recent good article, in French:
https://linuxfr.org/news/deces-de-bram-moolenaar-createur-de-vim

Another good one in Italian:
https://www.macitynet.it/bram-moolenaar/

**Comment by moehuster** [comment]
Chinese forum website zhihu:
[https://www.zhihu.com/question/615835724](https://www.zhihu.com/question/615835724)
[https://www.zhihu.com/question/615863902](https://www.zhihu.com/question/615863902)

**Comment by Bob-YsPan** [comment]
This article is from the new media's website from Taiwan:

*程式設計師最常用的文字編輯器「Vim」之父去世！他的慈善軟體改變了很多人的一生*
The roughly translate to English is: *The father of the most commonly used text editor "Vim" by programmers has passed away! His charity software changed the lives of many people*
[https://www.techbang.com/posts/108621](https://www.techbang.com/posts/108621)

**Comment by zeertzjq** [comment]
An article by Justin M. Keyes:
https://neovim.io/news/2023/08

**Comment by orsenthil** [comment]
This is my post - https://senthil.learntosolveit.com/posts/2023/08/09/thank-you-bram-moolenaar.html

**Comment by LunarWatcher** [comment]
Also not an article, but from vi.meta.SE: https://vi.meta.stackexchange.com/q/1606/21251

**Comment by rubileandro** [comment]
Here is an English article from DayFR Euro: https://euro.dayfr.com/trends/639418.html

**Comment by gonzaru** [comment]
Spanish article from atareao.es: https://atareao.es/podcast/homenaje-al-creador-de-vim-bram-moolenar/

**Comment by tobru** [comment]
We dedicated a vshn.timer to Brams work https://www.vshn.ch/blog/vshn-timer-194-in-memoriam-bram-moolenaar/

**Reply by 7r3y** [reply]
Hungarian article from hup.hu: https://hup.hu/cikkek/20230806/elhunyt_a_vim_megalkotoja_bram_moolenaar

**Comment by romoloman** [comment]
Italian article from miamammausalinux.org (translated my mother uses linux) a web site dedicated to linux and open source world.

https://www.miamammausalinux.org/2023/08/e-morto-alleta-di-62-anni-bram-moolenaar-creatore-delleditor-open-source-vim-e-pioniere-dello-sviluppo-charityware/

In this article are also highlighted the efforts of Bram for ICCF and UGANDA childrens.

**Comment by othree** [comment]
My article in Chinese https://blog.othree.net/log/2023/08/11/vim-boss-passed-away/

**Comment by chrisbra** [comment]
An article about Brams work on Zimbu: https://thenewstack.io/vim-creator-bram-moolenaars-forgotten-programming-language-zimbu/

**Comment by scavin** [comment]
This article(Chinese) is from Appinn.com: https://www.appinn.com/vim-developer-bram-moolenaar-gone/

**Comment by dnhn** [comment]
An article from Vietnam:
https://vnreview.vn/thread/vinh-biet-tac-gia-cua-vim-va-la-nguoi-hung-cua-nhieu-nha-phat-trien.70368744842734
